### ISSUE TITLE
Unable to type characters '[' and ']'

### ISSUE THREAD

**Issue by jgoux** [cerebral/webpackbin#3]
Hi,

In the new version of webpackbin, I'm unable to type `[` and `]`.

They're mapped to the shortcut which switch files. 😅

**Comment by bfitch** [comment]
@jgoux Sorry about that. What OS/browser are you using? And are you on: https://www.webpackbin.com/ ? 

Just tried it out using OSX with Chrome and it's working fine over here.

**Comment by jgoux** [comment]
I'm on Windows/Chrome, and to produce a `[` character I need to press the `alt gr+[` keys on my azerty keyboard.

The `alt gr` key produces the `alt` and `ctrl` keycodes at the same time, so I assume these [shortcuts](https://github.com/cerebral/webpackbin/blob/master/src/main.js#L48-L49) are fired.

Edit : Seems like a common issue! https://github.com/TryGhost/Ghost/issues/1148

**Comment by christianalfoni** [comment]
@bfitch @jgoux I removed the shortcut for now. Have another go if you want to @bfitch ;-)

**Comment by henri-hulski** [comment]
Think this is a problem with non English keyboards. I have also to press the `ALT GR` key to get [ or ] on a German keyboard.

**Comment by bfitch** [comment]
Good call. Best to remove it for now....I shall miss it though :(

**Comment by jgoux** [comment]
Why not just remap it to another shortcut ?

Like `ctrl + leftarrow` or `ctrl + b` to go back and `ctrl + rightarrow` or `ctrl + n` to go next ?

**Comment by bfitch** [comment]
Yeah, good idea. Now I am just paranoid that it will break something else 😧

**Comment by christianalfoni** [comment]
Haha, no worries... I have made lots of mistakes in production. Never feel bad about that :)
